core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -1,11 +1,25 @@
+com.google.common.collect.AbstractListMultimap
+com.google.common.collect.AbstractMultimap
 com.google.common.collect.AbstractSetMultimap
+# Artifactory - https://issues.jenkins-ci.org/browse/JENKINS-48991
+com.google.common.collect.ArrayListMultimap
 com.google.common.collect.EmptyImmutableList
 com.google.common.collect.EmptyImmutableSet
 com.google.common.collect.EmptyImmutableSortedSet
 com.google.common.collect.HashMultimap
+# First hit: https://github.com/jenkinsci/pitmutation-plugin/
+com.google.common.collect.HashMultiset
 com.google.common.collect.ImmutableList
+# Registered in XStream2 converters
+com.google.common.collect.ImmutableMap
+# Registered in XStream2 converters
+com.google.common.collect.ImmutableSet
```

Comment:
Is this actually required based on some experience? See below.

Reply:
No. I just assumed it may be still used somewhere since there are converters defined directly in the core. E,g, https://github.com/jenkinsci/jenkins/blob/08def67a18eee51de9f3f99bc2a792fee1c160e0/core/src/main/java/hudson/util/xstream/ImmutableSetConverter.java

Reply:
Well the converter says

```java
return ImmutableSet.class.isAssignableFrom(type);
```

It is `abstract`, so probably this would only affect usages from Remoting.